docs/flavors/developing.md

```diff
@@ -0,0 +1,27 @@
+# Developing your own flavor
+
+We welcome new flavors to the `cnc` project. Here's how we suggest you develop them.
+
+## Plan your flavor
+
+Make sure that it makes sense to be adding a flavor vs. modifying the exsting ones or customizing in your own project.
+
+A flavor should be a holistic reference architecture that uses a set of cloud services in convert to achieve a certain set of trade-offs between: cost, comliance, secutiry, complexity. For example, here are different flavors on AWS:
```

Comment:
Some typos here:

A flavor should be a holistic reference architecture that uses a set of cloud services in concert* to achieve a certain set of trade-offs between: cost, compliance*, security*, complexity. For example, here are different flavors on AWS: